lms/envs/common.py

```diff
@@ -1244,6 +1244,13 @@
     'js/vendor/url.min.js',
     'common/js/vendor/underscore.js',
     'js/vendor/underscore.string.min.js',
+
+    # Make some edX UI Toolkit utilities available in the global "edx" namespace
+    'edx-ui-toolkit/js/utils/global-loader.js',
```

Comment:
Reviewers: it is slightly odd that these edX files are added to the vendor bundle. However, they have to be loaded before RequireJS, and a few other approaches I took failed. Maybe I should rename the bundles to make it clearer what they are... any thoughts?